src/validators/redteam.ts

```diff
@@ -175,6 +176,8 @@ export const RedteamGenerateOptionsSchema = z.object({
   write: z.boolean().describe('Whether to write the output'),
   burpEscapeJson: z.boolean().describe('Whether to escape quotes in Burp payloads').optional(),
   progressBar: z.boolean().describe('Whether to show a progress bar').optional(),
+  liveRedteamConfig: z.any().optional().describe('Live redteam configuration from cloud'),
+  loadedFromCloud: z.boolean().optional().describe('Whether the config was loaded from cloud'),
```

Comment:
where and how are these used?

Reply:
I've gone ahead and consolidated these into a single variable, `configFromCloud`, which the remote configuration is stored in.  It's used to write the config the filesystem à la https://github.com/promptfoo/promptfoo/blob/feaa4a8f9ab6f529bd0dcdcadc89ba968a34f00f/src/redteam/shared.ts#L58